FlowCrypt/Functionality/Services/EnterpriseServerApi.swift

```diff
@@ -99,12 +114,24 @@ class EnterpriseServerApi: EnterpriseServerApiType {
                 method: .get,
                 body: nil
             )
-            let response = try? awaitPromise(URLSession.shared.call(request))
+            guard let safeReponse = try? awaitPromise(
+                URLSession.shared.call(
+                    request,
+                    tolerateStatus: Constants.getToleratedStatuses
+                )
+            ) else {
+                reject(EnterpriseServerApiError.parse)
+                return
+            }
+
+            if Constants.getToleratedStatuses.contains(safeReponse.status) {
+                resolve(.empty)
+            }
+
```

Comment:
We should only tolerate the error scenarios when calling `https://fes.<domain>/api/` endpoint specifically, because at that point we don't know if there is any server running there or not.

Don't tolerate any errors when calling `https://fes.\(userDomain)/api/v1/client-configuration` or any other APIs, because once we are calling those, we know that there was supposed to be FES running there (based o successful `https://fes.<domain>/api/` response earlier)